src/expr.rs

```diff
@@ -1318,18 +1318,21 @@ impl Rewrite for ast::Arm {
 
         let pats_str = format!("{}{}", pats_str, guard_str);
 
-        let body = match body.node {
+        let (mut extend, body) = match body.node {
             ast::ExprKind::Block(ref block) if !is_unsafe_block(block) &&
                                                is_simple_block(block, context.codemap) &&
                                                context.config.wrap_match_arms => {
                 if let ast::StmtKind::Expr(ref expr) = block.stmts[0].node {
-                    expr
+                    (false, &**expr)
                 } else {
-                    &**body
+                    (false, &**body)
                 }
             }
-            _ => &**body,
+            ast::ExprKind::Call(_, ref args) => (args.len() == 1, &**body),
+            ast::ExprKind::Closure(..) => (true, &**body),
```

Comment:
I would like to make it clear, should we use block indent for tuples when `fn_call_style = Block` and format like function call? (c.f. https://github.com/rust-lang-nursery/fmt-rfcs/issues/64)
```rust
let a = (
    xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx,
    yyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyy,
);
```